mbedtls/include/toit_config.h

```diff
@@ -2645,7 +2627,7 @@
  * The CTR_DRBG generator uses AES-256 by default.
  * To use AES-128 instead, enable \c MBEDTLS_CTR_DRBG_USE_128_BIT_KEY above.
  *
- * AES support can either be achieved through builtin (MBEDTLS_AES_C) or PSA.
+ * AES support can either be achived through builtin (MBEDTLS_AES_C) or PSA.
```

Comment:
wat

Reply:
same:
I'm taking the original file and only applying the configuration changes that are necessary. It's not worth fixing the typos every time.